core/controllers/skill_editor.py

```diff
@@ -199,11 +199,9 @@ def put(self, skill_id):
     def delete(self, skill_id):
         """Handles Delete requests."""
         skill_domain.Skill.require_valid_skill_id(skill_id)
```

Comment:
Add try expect else this will lead to 500* (Maybe in a separate PR?)

Reply:
Ok Noted. Will do in the next PR.

Reply:
Wait -- that seems wrong right? In general we should not use try/except to shield from 500s.

Instead we should check for validity and raise a 400 if wrong (using a method that doesn't raise when an error is found).

The practice of using try/except to shield from 500s has caused problems in the past, notably since we cannot figure out what code led to the error due to the stacktrace below the try/except being hidden.